Source/WebCore/Modules/WebGPU/GPUTexture.cpp

```diff
@@ -32,6 +32,11 @@
 
 namespace WebCore {
 
+GPUTexture::~GPUTexture()
+{
+    destroy();
```

Comment:
I actually think this is construing semantics. The API-exposed `destroy()` functions are supposed to mean "the browser's object is alive - and can continue to be interacted with, methods called on it, validation is performed, state tracking is still performed, etc, but there's just no Metal object inside.

That's different than what you're doing here, which is trying to communicate "the browser object doesn't need to exist any more at all" between processes, to avoid leaks. I think we should treat those 2 things as distinct, because I believe they have observably different semantics.